### ISSUE TITLE
girb bash executable throws ruby error

### ISSUE THREAD

**Issue by Nakilon** [AndyObtiva/glimmer-dsl-tk#3]
```none
$ girb
/Users/nakilon/.rbenv/versions/2.3.8/bin/girb:22:in `load': /Users/nakilon/.rbenv/versions/2.3.8/lib/ruby/gems/2.3.0/gems/glimmer-dsl-tk-0.0.6/bin/girb:5: syntax error, unexpected tGVAR, expecting keyword_do or '{' or '(' (SyntaxError)
/Users/nakilon/.rbenv/versions/2.3.8/lib/ruby/gems/2.3.0/gems/glimmer-dsl-tk-0.0.6/bin/girb:7: syntax error, unexpected '~', expecting keyword_do or '{' or '('
if !(test -f ~/.girb_runner_exit); then
              ^
unmatched close parenthesis: /.girb_runner_exit); then
  rm -rf ~/
/Users/nakilon/.rbenv/versions/2.3.8/lib/ruby/gems/2.3.0/gems/glimmer-dsl-tk-0.0.6/bin/girb:10: syntax error, unexpected end-of-input, expecting ')'
	from /Users/nakilon/.rbenv/versions/2.3.8/bin/girb:22:in `<main>'
```
macOS 11.4

**Comment by AMaleh** [comment]
Sorry, but the project assumes RVM not rbenv, and it was a quick exploratory early alpha written on macOS 10.15.7.

**Comment by AMaleh** [comment]
Just fixed girb in v0.0.7 by turning it into a pure Ruby script (instead of bash).
